edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -1,684 +1,150 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-use std::env;
 use std::ffi::OsString;
 use std::fs::File;
-use std::io::{Cursor, Seek};
+use std::io::Seek;
+use std::io::SeekFrom;
+use std::io::Write;
+use std::io::{Cursor, Read};
```

Comment:
What is our style here? Grouped imports or individual like for each?

Reply:
fixed